assets/styles/utilities/_states.scss

```diff
@@ -63,9 +61,9 @@
 //     }
 // }
 
-// /**
-//  * Display a hidden-by-default element.
-//  */
+// //
+// // Display a hidden-by-default element.
+//
```

Comment:
Double comment.

```suggestion
//
// Display a hidden-by-default element.
```

Reply:
It's because the whole block is commented, to avoid an error on uncommenting.
Should I remove the double comment anyway ?

Reply:
In which case, the commented block should follow across the snippets that disabled. For example, to me this is much more clear why the block comment is commented:

```scss
// //
// // Display a hidden-by-default element.
//
// .is-shown {
//     @include u-shown;
// }
//
// table.is-shown {
//     display: table !important;
// }
//
// tr.is-shown {
//     display: table-row !important;
// }
//
// td.is-shown,
// th.is-shown {
//     display: table-cell !important;
// }
```